docs/upgrade/xenial_backup_install_restore.rst

```diff
@@ -0,0 +1,301 @@
+of your existing Ubuntu 14.04 (Trusty) instance, install a new instance based on
+Xenial, and restore the data from the backup to the Xenial-based installation.
+
+You can follow this procedure on your existing hardware (causing significant
+downtime), or you can install using redundant hardware and perform a cut-over
+once you have validated that the new installation is working (minimizing
+downtime).
+
+.. note ::
+  This procedure requires physical access to the servers and firewall
+  to complete the installation. If you do not have physical access to the
+  hardware, consider using the method documented in :doc:`xenial_upgrade_in_place`,
+  which can be completed from the *Admin Workstation* over Tor.
+
+
+Before you begin
+----------------
+
+Before you get started, make sure that you have completed the :doc:`preparatory
+steps for the Xenial upgrade <xenial_prep>`. You will need physical access to
+the servers and firewall, and you will need a monitor and keyboard to connect to
+the servers during the OS installation.
+
+If you are planning to install the Xenial instance on new hardware, make sure
+that you have all the necessary hardware on hand and configured. This guide
+assumes that you are either using the existing hardware firewall, or a new
+firewall configured with the same settings as your existing one.
+
+Although downtime should be minimal, you should coordinate the timing of the
+upgrade with the people responsible for checking for submissions. If they have
+ongoing conversations with sources, or are expecting submissions, it may be
+necessary to reschedule the upgrade. Consider communicating the planned downtime
+and the reason for it on your instance's *Landing Page*.
+
+Step 1: Back up your instance
+-----------------------------
+
+Start up the *Admin Workstation* with persistence enabled and an administration
+password set. Then, once you are connected to the Tor network, open a terminal
+and run the following commands:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop
+ ./securedrop-admin setup
+ ./securedrop-admin backup
+
+This will create a backup named ``sd-backup-<date>.tar.gz`` in the
+``~/Persistent/securedrop/install_files/ansible-base`` directory. Make a note of
+the exact name, as you'll need it later.
+
+.. note:: 
+ The backup files are not encrypted by default. If you copy them to a USB stick
+ or other media you should ensure that the device uses encrypted storage, or
+ that you encrypt the backup files themselves using a strong encryption method. 
+
+You should also make a copy of the configuration files that allow the *Admin
+Workstation* to connect to the hidden services on the *Application Server*. To
+do so, run the following commands from the terminal:
+
+.. code:: sh
+
+  cd ~/Persistent/
+  mkdir app_service
+  cp securedrop/install_files/ansible-base/app-*ths app_service/
+  cp securedrop/install_files/ansible-base/site-specific app_service/
```

Comment:
isn't the path `install_files/ansible-base/group_vars/all/site-specific` by default?